bazaarvoice/emodb
# tests for /bus/1/{subscription}/ack

## What Are We Doing Here?

We have added postman tests for /bus/1/{subscription}/ack
## How to Test and Verify

1. Check out this PR
2. Import new postman collection to postman, Run collection
3. Newly added tests are passed. Old tests are also pass without failure.

## Risk
No risk, as this is new code which is not dependent with existing code
### Level 

`Low`

### Required Testing

`Smoke`

### Risk Summary

Add one or a few complete sentences about the possible risks or concerns for
this change.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
